theme/components/buttonsTheme.ts

```diff
@@ -0,0 +1,117 @@
+import type { ComponentStyleConfig } from '@chakra-ui/theme'
+import { mode } from '@chakra-ui/theme-tools'
+import { NAMED_COLORS } from 'styles/constants'
```

Comment:
Merge last changes from `master` and use `theme/constants` instead of `styles/constants`